run_selenium_system_test.sh

```diff
@@ -9,7 +9,7 @@ tags    - Space separated list of tags. Only run tests with the given
           tags. Not specifying any tags will run all tests.
```

Comment:
On line 6 of  run_selenium_system_test.sh can we include [zap-chrome], please?

Reply:
Yeah sure